style-gutenburg.scss

```diff
@@ -3,5 +3,6 @@
     @import 'node_modules/govuk-frontend/govuk/tools/px-to-rem';
     @import 'node_modules/govuk-frontend/govuk/settings/media-queries';
     @import 'node_modules/govuk-frontend/govuk/helpers/media-queries';
+    @import 'src/**/gutenberg/*.scss';
     @import 'src/**/*.scss';
```

Comment:
Doesn't `@import 'src/**/*.scss';` reach inside `gutenburg/`?

Reply:
Thanks for asking because this change will affect the whole plugin and blocks. The way we had it was that everything gets compiled into one...SASS for the frontend and backend. However, my block needed to have separate CSS on the backend to the frontend. So the way I've approached this is that now every block will have a gutenberg folder with specific SASS aimed at only the backend. If you want to use the same SASS for both, you just have to include an `@import` into your gutenberg sass that points to the frontend sass.

Reply:
Ah :) normally you'll see `block/editor.scss` and `block/style.scss` without a need to say the word gutenburg. Have a look in the Atomic Block plugin and see how they've done it. I really like the way they've approached it :o)

Reply:
Okay, I see and then just have `@import 'src/**/editor.scss';` and `@import 'src/**/styles.scss';` . Yes, that would work to. I like the simplicity. I went with the folder approach in case you had a plugin where you wanted to break the SASS into may files, but in the above approach I guess you could do editor-colours.scss or something like that.

Reply:
Perfect